client/firebase/initNewGame.js

```diff
@@ -89,7 +98,8 @@ const generateStacksForPlayer = (playerNum) => {
   return {
     [`p${playerNum}BigStack`]: cards.slice(0, 35),
     [`p${playerNum}DrawnStack`]: false, // placeholder val for firebase
-    [`p${playerNum}LittleStack`]: cards.slice(35,48),
+    // [`p${playerNum}LittleStack`]: cards.slice(35,48),
+    [`p${playerNum}LittleStack`]: [],
```

Comment:
I believe this needs to be put back, right?

Reply:
Correct, we will address